project/SparkSnowflakeBuild.scala

```diff
@@ -45,9 +45,9 @@ object SparkSnowflakeBuild extends Build {
       organization := "net.snowflake",
       scalaVersion := sys.props.getOrElse("SPARK_SCALA_VERSION", default = "2.11.7"),
       crossScalaVersions := Seq("2.10.5", "2.11.7"),
```

Comment:
Spark 2.3.0 drops support for Scala 2.10.x --
See `Other notable changes`: https://spark.apache.org/releases/spark-release-2-3-0.html
Which links to: https://issues.apache.org/jira/browse/SPARK-19810

Reply:
Thanks, @ahepburn. Good catch.